onnxruntime/core/providers/cuda/tensor/space_depth_ops.cc

```diff
@@ -173,11 +173,11 @@ Status SpaceToDepth<Layout>::ComputeInternal(OpKernelContext* context) const {
   int64_t output_width = -1;
 
   ORT_RETURN_IF_ERROR(
-      InputValidationsAndOutputDimsCalc<Layout == LAYOUT_NHWC>(input,
-                                                               batch,
-                                                               input_depth, input_height, input_width,
-                                                               output_depth, output_height, output_width,
-                                                               true));
+      InputValidationsAndOutputDimsCalc < Layout == LAYOUT_NHWC > (input,
```

Comment:
this line becomes more confusing after formatted. Should we use
```
InputValidationsAndOutputDimsCalc<(Layout == LAYOUT_NHWC)>( ...
```

Reply:
It is autogenerated. Otherwise we need to adjust the `.clang-format` config file to avoid such changes.

Reply:
It might because clang-format could not recognize this is a template. It thought the character '<' is an operator.